examples/default.yaml

```diff
@@ -2,17 +2,16 @@
 # BASIC CONFIGURATION
 # ===================================================================== #
 
-# Default values are specified by `null` instead of the builtin default value,
-# so they can be overridden by the default.yaml mechanism documented at the
-# end of this file.
+# Default values in this YAML file are specified by `null` instead of Lima's "builtin default" values,
+# so they can be overridden by the $LIMA_HOME/_config/default.yaml mechanism documented at the end of this file.
 
 # Arch: "default", "x86_64", "aarch64".
-# Default: "default" (corresponds to the host architecture)
+# 🟢 Builtin default: "default" (corresponds to the host architecture)
 arch: null
 
-# An image must support systemd and cloud-init.
-# Ubuntu and Fedora are known to work.
-# Default: none (must be specified)
+# OpenStack-compatible disk image.
+# 🟢 Builtin default: null (must be specified)
+# 🔵 YAML default: Ubuntu 21.10 Impish Indri images
```

Comment:
"YAML default" might be still confusing, but can't come up with a better terminology

Reply:
Yeah, I don't know, "YAML default:" sounds weird to me.

Maybe "This file:"?

Still weird, but at least makes sure that it is the value from this specific file, and not a generic "YAML default" (what does that even mean?). Like, if there is a "YAML default", could there be a different "JSON default", or "commandline default", or what?

Reply:
Changed to "This file"